src/Main.elm

```diff
@@ -0,0 +1,73 @@
+module Main exposing (..)
+
+import Browser
+import Html exposing (Html, text, div, h1, img)
+import Html.Attributes exposing (src)
+
+
+---- MODEL ----
+
+type alias UserInfo = 
+    { name : String
+    , age: Int
+    }
+
+type User = 
+    Anonymous
+    | Authenticated UserInfo
+
+type alias Model =
+    UserInfo
```

Comment:
This won't work - as we said earlier it should be `Model = User` or `Model = {user : User}`

Reply:
I changed it to User